.github/licenserc.yaml

```diff
@@ -14,7 +14,8 @@ header:
     - "apis/**"
     - "assets/**"
     - ".npmignore"
-    - "**/*.npmignore"
+    - ".npmignore"
+    - ".prettierrc"
```

Comment:
This extension is alredy added to the end of the list

Reply:
Good catch